packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 - Add `defaultDnt` to support setting the dnt flag for SLAS. Upgrade `commerce-sdk-isomorphic` to v3.1.1 [#1979](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1979)
```

Comment:
This branch seems to contain the DNT changes, and the changelog has 3.1.0 in it. 🤔

Reply:
You're right. I branched off `develop` instead of the latest release. Let me re-create the PR